src/txn/txn.c

```diff
@@ -308,6 +309,21 @@ __wt_txn_get_snapshot(WT_SESSION_IMPL *session)
     __txn_get_snapshot_int(session, true);
 }
 
+/*
+ * __wt_swap_snapshot --
+ *     Swap the snapshot pointers.
+ */
+void
+__wt_swap_snapshot(uint64_t **snap_a, uint64_t **snap_b)
```

Comment:
Does this need to be with __wt prefix?

Reply:
Nice catch, thanks @quchenhao. Fixed in 074dc0fde461582847671b3de4080b87a9f331f7. Changed the function name to `__txn_swap_snapshot`